modules/graphics/enums/CullMode.lua

```diff
@@ -0,0 +1,18 @@
+return {
```

Comment:
The enum has to be required in the module:

https://github.com/love2d-community/love-api/blob/master/modules/graphics/Graphics.lua#L3540